### DISCUSSION TITLE
Customize location where style tag is injected? (useful for shadow DOM)

### DISCUSSION THREAD

**Discussion by vedantroy** [tw-in-js/twind#332] [Q&A]
Is it possible to customize where the style tag will be injected? For example, if I'm running a React app inside of a shadow DOM.

Use case: https://dev.to/tryeladd/preact-in-the-shadow-dom-ao8

**Comment by sastan** [comment]
Would the new web component package solve your problem (https://github.com/tw-in-js/twind/issues/386#issuecomment-1353417598)?
